### PR TITLE
Client side

### PR DESCRIPTION
**Pull request by shreyaa-s-zz**

### CODE REVIEW

**Comment by karan0299** [comment]
As a good practice , always provide some description about the PR

File: IRequest.php [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<?php
+interface IRequest
+{
+    public function getBody();
+}
```
Comment (karan0299) [inline IRequest.php]: insert new line at the end

File: html/pastorder.php [hunk 2]
```diff
@@ -12,25 +11,145 @@
...
+                        alert("Feedback added!");
+                        //$('form input[type="button"]').prop("disabled", true);
+                        //this.disabled = true;
+                        //$("#submitbutton").attr("disabled", true);
+                    } else {
+                        alert("Connection Error");
+                    }
+                }
+            })
+        }); */
```
Comment (karan0299) [inline html/pastorder.php]: remove unnecessary comments

**Review by karan0299 — CHANGES_REQUESTED** [review]
Perform the required changes , rest  LGTM

**Comment by shreyaa-s-zz** [comment]
@karan0299 done!

**Comment by karan0299** [comment]
Have you done testing?

**Pull request merged**